sdk/servicebus/azure-servicebus/azure/servicebus/aio/_servicebus_receiver_async.py

```diff
@@ -214,38 +214,33 @@ def __init__(
         self._receive_context = asyncio.Event()
         self._handler: ReceiveClientAsync
 
-    # Python 3.5 does not allow for yielding from a coroutine, so instead of the try-finally functional wrapper
-    # trick to restore the timeout, let's use a wrapper class to maintain the override that may be specified.
-    class _IterContextualWrapper(collections.abc.AsyncIterator):
```

Comment:
removing this as we no longer are required to support 3.5 or 3.6 so we can make our own async generator